Tests/XMLCoderTests/RootLevetExtraAttributesTests.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+@testable import XMLCoder
+
+class RootLevetExtraAttributesTests: XCTestCase {
```

Comment:
```suggestion
final class RootLevetExtraAttributesTests: XCTestCase {
```

Reply:
i completely agree with this, but i checked the other classes and so i followed the same. changed 😊 👍

Reply:
Sorry about that, the library does have some inconsistencies which is caused more by the lack of my attention than anything else 🙈

Reply:
that's normal 🤷 i should have followed what i usually do, but i prefer to respect the "style" 😅